lsr_role2collection.py

```diff
@@ -1615,6 +1665,27 @@ def process_readme(
             env=env,
         )
 
+    # Handle --extra-mapping FQCN0:FQCN1 or FQCN2:ROLE3
+    file_patterns = ["*.yml", "*.md", "*.py"]
```

Comment:
`.py` files?

Reply:
I thought there could be some slightest possibility, but no such thing... :p
If so, we don't have to check files under the plugin_dir.
https://github.com/linux-system-roles/auto-maintenance/pull/155/files#diff-f5898ea097abc3a298c8aa8850d649247348992056a612047ca43828e0b7a56aR1682-R1683

Reply:
I agree with you. :) I'm removing them...